proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -264,15 +264,15 @@ message LoginPasswordlessRequest {
 }
 
 message FileTransferRequest {
-  string cluster_uri = 1;
+  // login is the SSH username to use for the transfer.
```

Comment:
After you remove a field, mark it as `reserved` so no one will reuse the number in the future.